source/agora/consensus/protocol/Nominator.d

```diff
@@ -220,10 +226,14 @@ extern(D):
         this.acceptBlock = externalize;
     }
 
-    /// Shut down the envelope processing timer
+    /// Shut down the timers
     public void shutdown () @safe
```

Comment:
Who calls this? I couldn't find any references

Reply:
Your right it isn't called from anywhere. I will add it to node shutdown.

Reply:
Added a commit to improve shutdown.

Reply:
Now in https://github.com/bosagora/agora/pull/3056